game/src/test/java/net/demilich/metastone/tests/util/TestBase.java

```diff
@@ -521,4 +526,36 @@ protected static Minion playMinionCard(GameContext context, Player player, Card
 		return getSummonedMinion(player.getMinions());
 	}
 
+	protected void printAll(GameContext context, Player player) {
```

Comment:
Printing can be handy but it'll break CI.

If you need to inspect something, start IntelliJ, import project, choose Gradle project (gives you a working project), then set the breakpoint and click debug on the specific test's little play icon:

<img width="891" alt="screenshot 2018-06-26 12 11 55" src="https://user-images.githubusercontent.com/2229300/41933937-22719afc-793a-11e8-8b77-1348981fa396.png">